src/User.java

```diff
@@ -0,0 +1,78 @@
+import java.util.Scanner;
+
+
+public class User {
+    private String userName;
+    private String action;
```

Comment:
Action is not information that really needs to be tracked over the life of the object.  It should be localized to the method that actually processes the action they want.